common/components/lua-script.cpp

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2013-2016 Trillek contributors. See AUTHORS.txt for details
 // Licensed under the terms of the LGPLv3. See licenses/lgpl-3.0.txt
 
-#include "components/lua-script.hpp"
+#include "lua-script.hpp"
```

Comment:
are these changes required by a change to the build system? or were both paths valid before? because I definitely don't think that both paths should be valid...

Reply:
The header is located relative to the source, so both paths are valid and both were valid.